### PR TITLE
Initial version of the orb

### PR DESCRIPTION
**Pull request by suryasr007**
- Added the orb file.
- CircleCI personal tokens are visible in the logs. Hence not publishing automatically until it is resolved (https://ideas.circleci.com/ideas/CCI-I-397).

Ref: https://github.com/niteoweb/operations/issues/1111

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,67 @@
...
+```bash
+$ make dev-release tag=1.0.1
+```
+
+> Note: It is a good idea to test a dev version of the orb first before publishing new public version.
+
+
+## Publishing Orb
+
+* Make sure you have privileges to publish a orb to production.
```
Comment (zupo) [inline README.md]: an orb

File: README.md [hunk 2]
```diff
@@ -1,2 +1,67 @@
...
+* Once the dev version is tested, You can publish a new orb version manually, by specifying new version:
+```bash
+$ make prod-release tag=1.0.1
+```
+> Note: Once the new version of orb is published, The same can be tagged in the github releases.
```
Comment (zupo) [inline README.md]: `, The same` -> capital letter?

File: src/orb.yml [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+  format_test:
+    description: Test if the files are formatted
+    usage:
+      version: 2.1
+      orbs:
+        untracked_changes: niteo/check-untracked-changes@1.0.0
+      jobs:
+        test:
+          docker:
+            - image: circleci/python:3.7  #Can be any executor
```
Comment (zupo) [inline src/orb.yml]: `#Can` -> missing space

File: README.md (hunk 1 above)
Comment (suryasr007) [reply README.md]: Missed it. Thanks 👍

**Pull request merged**